### PR TITLE
feat: export utils from core for other packages

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**

feat

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
NA

**Summary**
Exported some utils which are used by other packages so we won't depend on the file structure as it can change in core hence it will break other packages.

**Does this PR introduce a breaking change?**
No

**Other information**
NA

### CODE REVIEW

**Review by evilebottnawi — COMMENTED** [review]
Can you remove `template`?

**Comment by anshumanv** [comment]
which template @evilebottnawi ?

**Comment by evilebottnawi** [comment]
From the PR :smile:

File: packages/webpack-cli/lib/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+const { commands } = require('./utils/cli-flags');
+const logger = require('./utils/logger');
+const getPackageManager = require('./utils/get-package-manager');
+
+module.exports = WebpackCLI;
+
+// export additional utils used by other packages
+module.exports.logger = logger;
+module.exports.commands = commands;
+module.exports.getPackageManager = getPackageManager;
```
Comment (evilebottnawi) [inline packages/webpack-cli/lib/index.js:11]: Let's export this under `util`, like `webpackCli.util.logger`?

**Review by evilebottnawi — COMMENTED** [review]
Let's do small improvement

**Review by evilebottnawi — APPROVED** [review]
/cc @webpack/cli-team

**Review by snitin315 — APPROVED** [review]

**Pull request merged**